lib/common/tests/xml/Makefile.am

```diff
@@ -14,7 +14,6 @@ include $(top_srcdir)/mk/unittest.mk
 # Add "_test" to the end of all test program names to simplify .gitignore.
 check_PROGRAMS = \
 		 pcmk__xml_escape_test		\
-		 pcmk__xml_init_test		\
```

Comment:
Also add `/lib/common/tests/xml/pcmk__xml_init_test` to `.gitignore`.

Reply:
`.gitignore` already includes `/lib/*/tests/*/*_test`

Reply:
Ah right, so it does.